### DISCUSSION TITLE
无权访问X-lab的语雀空间

### DISCUSSION THREAD

**Discussion by thredreams** [X-lab2017/open-wonderland#217] [6️⃣ 提问题] [answered]
我对你们的项目有些感兴趣，所以想看看相关内容，结果发现X-lab的语雀里的很多资源都是不开放的，我想知道，这是意外还是出于某些考虑刻意这么设计的？

**Comment by zhicheng-ning** [comment, marked as answer]
语雀 X-lab 内部的资源是不对外开放的~，对我们项目感兴趣的话，可以前往相应的仓库参加每周例会的讨论
